qiskit/visualization/state_visualization.py

```diff
@@ -710,8 +710,26 @@ def plot_state_qsphere(
            qc.h(0)
            qc.cx(0, 1)
 
-           state = Statevector.from_instruction(qc)
+           state = Statevector(qc)
            plot_state_qsphere(state)
+        .. jupyter-execute::
+
+           from qiskit import QuantumCircuit
+           from qiskit.quantum_info import DensityMatrix
+           from qiskit.visualization import plot_state_qsphere
+           import numpy as np
+           %matplotlib inline
```

Comment:
I don't think this is needed the jupyter execute directives in a document share a kernel. (also I'm not sure we need it generally, but regardless just having a single magic in the first cell in the page is sufficient)

Reply:
I am not sure %matplotlib inline is needed either. @Guillermo-Mijares-Vilarino can you try to remove this and see if the plots are still rendered as expected?

Reply:
Sure!

Reply:
Removed `%matplotlib inline` and the docs were built successfully.